src/components/_variables.scss

```diff
@@ -8,6 +8,7 @@ $font-family: "DINOT", sans-serif !default;
 $white: #fff;
 $grey: #ccc !default;
 $charcoal: #3a333c !default;
```

Comment:
Let's check with Antoine although I think `charcoal` could be replaced with `slate` in all components